tests/unit-fail-compilation/dup_params1.ref

```diff
@@ -1,7 +1,7 @@
 File: dup_params1.c
 Line: 1
-Error: CV-FD1
-Description: duplicate parameters in function definition.
+Error: CV-TDL6
```

Comment:
It still gets triggered in the case of old style defs with duplicate ids in
the id list (but not duplicate declarations). So many caveats...
On Nov 10, 2015 12:24 PM, "dwightguth" <notifications@github.com> wrote:

> In tests/unit-fail-compilation/dup_params1.ref
> <https://github.com/kframework/c-semantics/pull/166#discussion_r44443811>:
>
> > @@ -1,7 +1,7 @@
> >  File: dup_params1.c
> >  Line: 1
> > -Error: CV-FD1
> > -Description: duplicate parameters in function definition.
> > +Error: CV-TDL6
>
> I believe you will also need to update CV-FD1 in the examples/simple
> directory. Do you know if it's even possible to trigger this error anymore
> or has it been completely superceded?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kframework/c-semantics/pull/166/files#r44443811>.
>